jac-uk/qt
# amend instructions for scenario

## What's included?
Amended instructions for scenario tests
Closes #38

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Go to a scenario test instructions page. Ensure you can see the new amended instructions

> Important information
> 
> • This test contains 1 Scenario: Scenario 1 with 3 questions [same line]
> • Question 1: 25 marks and 300-word limit
> • Question 2: 15 marks and 200-word limit
> • Question 3: 60 marks and 600-word limit
> • You have 600 minutes to complete this test
> • You must submit your test by 21:00 on 1 January 2024
> • Ensure your internet connection is stable before you start the test
> • If you experience any technical issues, please contact the helpdesk.
> • Once you have started the test, do not open the test on a second device
> • Your answers will be saved as you progress, but you will be able to edit them at the end before you submit your test, assuming you still have time remaining.
> 
> Additional instructions
> 
> • You must answer the questions, giving your full reasoning and referring to the relevant parts of the legislation, rules and other information provided to you both in the pre-reading and the test
> • The scenario test has been designed to test the following competencies: Exercising Judgement, Possessing and Building Knowledge, Assimilating and Clarifying Information, Working and Communicating with Others and Managing Work Efficiently

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

![image](https://github.com/jac-uk/qt/assets/44227249/b340c5db-9411-47c0-a7c4-2433fd774bed)

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
